src/pagingscrollview/src/NIPagingScrollView.m

```diff
@@ -694,6 +722,20 @@ - (void)setPageMargin:(CGFloat)pageMargin {
   [self setNeedsLayout];
 }
 
+- (void)setPageInset:(CGFloat)pageInset {
+  _pageInset = pageInset;
+  _scrollView.frame = [self frameForPagingScrollView];
```

Comment:
It seems odd for the page inset setter to modify the scroll view's frame. I would expect the setter to invalidate the layout like setPageMargin: above. On a related note I get the sense that this class is missing an implementation of layoutSubviews?